packages/lodestar/src/chain/forkChoice/interface.ts

```diff
@@ -18,6 +18,7 @@ export interface ILMDGHOST {
   headStateRoot(): Uint8Array;
   getJustified(): Checkpoint;
   getFinalized(): Checkpoint;
+  getAncestor(root: RootHex, slot: Slot): RootHex | null;
```

Comment:
Can the input/output be Uint8Array instead of hex string? Perhaps an outer function, public, with Uint8Array input/output, and an inner function, private, as before, with hex string input/output